File: app/src/test/java/com/hana053/micropost/testing/EmptyResponseBody.java
package com.hana053.micropost.testing;

import okhttp3.MediaType;
import okhttp3.ResponseBody;
import okio.BufferedSource;

public class EmptyResponseBody extends ResponseBody {
    @Override
    public MediaType contentType() {
        return null;
    }

    @Override
    public long contentLength() {
        return 0;
    }

    @Override
    public BufferedSource source() {
        return null;
    }
}

File: app/src/androidTest/java/com/hana053/micropost/testing/EmptyResponseBody.kt
package com.hana053.micropost.testing

import okhttp3.MediaType
import okhttp3.ResponseBody
import okio.BufferedSource

class EmptyResponseBody : ResponseBody() {
    override fun contentType(): MediaType? {
        return null
    }

    override fun contentLength(): Long {
        return 0
    }

    override fun source(): BufferedSource? {
        return null
    }
}
